### DISCUSSION TITLE
How can I get maple font look like this?

### DISCUSSION THREAD

**Discussion by blurm** [subframe7536/maple-font#290] [Q&A]
This is from discord, the ligatures are quite beautiful
![maplefont](https://github.com/user-attachments/assets/38fd8a77-1bcb-4f7c-9088-36ebb92af751)

Mine in vscode are quite different
![image](https://github.com/user-attachments/assets/261fe942-8c23-4039-8ea0-7a05ecb6aa1b)

Do I need extra settings for the ligatures?

**Comment by subframe7536** [comment]
The font in first picture is Maple Mono V7, which is still in beta (but almost stable)

**Comment by blurm** [comment]
> The font in first picture is Maple Mono V7, which is still in beta (but almost stable)

Thanks! I guess I will wait for the stable version

**Comment by indranildeveloper** [comment]
Hey, I have a question. How I can use the old `l` italic in v7? I really liked the old italic `l`.

**Reply by subframe7536** [reply]
Sorry but this is no longer supported. I think the overall style of the new version of `l` and font is more cohesive.

**Comment by asu4net** [comment]
Hi, I installed the default version of the V7 and I still have the old version of `l` on sublime text...

**Reply by subframe7536** [reply]
Ligature is limited in sublime text. https://www.sublimetext.com/docs/ligatures.html

You can custom build to freeze the feature or use Maple Mono Normal instead.
